target/linux/pistachio/base-files/lib/functions/mac.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+#
+# Copyright 2016 by Imagination Technologies Limited and/or its affiliated group companies.
+#
+# All rights reserved.  No part of this software, either
+# material or conceptual may be copied or distributed,
+# transmitted, transcribed, stored in a retrieval system
+# or translated into any human or computer language in any
+# form by any means, electronic, mechanical, manual or
+# other-wise, or disclosed to the third parties without the
+# express written permission of Imagination Technologies
+# Limited, Home Park Estate, Kings Langley, Hertfordshire,
+# WD4 8LZ, U.K.
+#
+# common shell lib to obtain wifi mac address
+
+
+get_primary_macaddr()
```

Comment:
at some places you are following function_name() { and at some places it is
function_name ()
{

make sure we follow the same guideline